source/module_elecstate/test/elecstate_lcao_parallel_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+np=`cat /proc/cpuinfo | grep "cpu cores" | uniq| awk '{print $NF}'`
```

Comment:
You can run `nproc` for it.

Reply:
but I need the physical core number, is there any command?

Reply:
I see. 
So, does abacus run faster on a process number equal to core number, or thread number?

Reply:
No. I have did some tests, when the number of physical cores is reached in parallel, the acceleration has reached the maximum. Beyond this number, there will be no additional acceleration effect.

Reply:
OK! One last question: how did you set the OMP thread number? A physical core have 2 threads, and setting the thread number to 2 might help? (just guessing, I'm not sure)

Reply:
In my previous testing, setting the  thread number to 2 has no obvous improve. I don't know if it is because ABACUS has few codes using OPENMP? May be we need a systemic testing.

Reply:
Great, thanks a lot!